### ISSUE TITLE
IndexOutOfRangeException Error

### ISSUE THREAD

**Issue by gtrof** [DotModGroup/ColdWaters-DotMod#136]
Hello, 

So, I installed the latest version of Dotmod and when I try to load a saved game (NATO 1984, Los Angeles Class SSN), I keep getting this error. This has happened with two different saved games. 

I've attached the screenshot of what I see, and I believe the correct Output Log. If you need more information, please let me know. 
[output_log.txt](https://github.com/DotModGroup/ColdWaters-DotMod/files/9835117/output_log.txt)

![Error](https://user-images.githubusercontent.com/116325337/197094802-63a50dea-ee14-4425-8a6f-091cdd30ac6b.png)

**Comment by HobbitJack** [comment]
Was the save loaded a DotMod save? Non DotMod saves are not compatible with DotMod.

**Comment by gtrof** [comment]
Hey there, yeah, I saved the DotMod game I had begun, and it wouldn't open.

**Comment by HobbitJack** [comment]
And one more question -- you can confirm that this save is from this version of the mod?

**Comment by gtrof** [comment]
Yeah, it was based on the latest version that was available in Oct (DotMod.v0.4a-2b, according to the zip file). I installed the new version, then started playing with it, and started getting this error trying to load the saved campaign I began under the new version.

**Comment by Craxelicrox** [comment]
I get the same when launching RUM-139 from the VLS launcher (ASW). Very annoying and hampers the game play exponentially. And pointers would be appreciated.
